src/css/main.scss

```diff
@@ -40,77 +37,307 @@ header {
   }
 }
 
-.wrapper {
-  max-width: 1280px;
-  margin: auto;
+.wrapper,
+.header {
+  margin-left: 1.11rem;
+  margin-right: 1.11rem;
 }
 
-h1, h2, h3, h4, h5, h6, p.serif {
-  font-weight: 100;
+.content-container {
+  display: flex;
+  margin-bottom: 5rem;
+  max-width: 1440px;
```

Comment:
Why this one using px value? Any specific reason?

Reply:
No particular reason, I changed to rem! :)